CHANGES_NEXT_RELEASE

```diff
@@ -1,3 +1,4 @@
+Fix: Listing service groups returns only a single service group (iota-ul #391)
```

Comment:
```suggestion
Fix: Listing service groups returns only a single service group (#894)
```

Af far as I understand, this issue is in the iotagent-node-lib librery, affecting not only IOTA-UL but other agents. Thus I have transferred it to library repo and issue number is now other.